crimsobot/cogs/games.py

```diff
@@ -618,11 +629,11 @@ def vote_check(msg: discord.Message) -> bool:
 
 
     @commands.command(hidden=True)
-    async def daily(self, ctx: commands.Context, lucky_number: int = 0) -> None:
+    async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
```

Comment:
Heck I forgot about one functionality change! `>daily` now *requires* you to guess a number, whereas before, if you did not, your "guess" was 0 and the winning number could never be zero. No purpose for that really, and besides it mucked up the code in `utils/games.py`